modules/dynamic-plugins/proc-export-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,116 @@
+[id="proc-export-third-party-plugins-rhdh_{context}"]
+= Exporting third-party plugins in {product}
+
+To use plugins in {product}, the plugins must be exported as derived dynamic plugin package. These packages contain the plugin code and dependencies, ready for dynamic plugin integration into {product-short}.
+
+The third-party plugins include:
+
+Backend plugins::
++
+--
+To ensure compatibility with the dynamic plugin support and enable their use as dynamic plugins, existing backend plugins must be compatible with the plugin instance. Additionally, these plugins must be rebuilt using a dedicated CLI command.
+
+The plugin instance is created using `createBackendPlugin()` or `createBackendModule()`. You must export the plugin instance as the default export from either the main package or an `alpha` package (if the plugin instance support is still provided using `alpha` APIs). This doesn't add any additional requirement on top of the standard plugin development guidelines of the plugin instance.
+
```

Comment:
The following paragraph is missing from here. It is important as it gives a high-level description of how his process works. 

```
The dynamic export mechanism identifies private, non-backstage dependencies, and sets the `bundleDependencies` field in the `package.json` file for them, so that the dynamic plugin package can be published as a self-contained package, along with its private dependencies bundled in a private `node_modules` folder.
```

Reply:
Do we need to mention non-backstage? 
I've added the following info:

`The dynamic export mechanism identifies private dependencies and sets the `bundleDependencies` field in the `package.json` file. This export mechanism ensures that the dynamic plugin package is published as a self-contained package, with its private dependencies bundled in a private `node_modules` folder.`